original/pong/main.py

```diff
@@ -181,13 +201,31 @@ def get(self, name):
                                     d['points_lost']))
                         for opponent, d in matchup_data.items()]
         template = JINJA_ENVIRONMENT.get_template('user.html')
-        self.response.write(
-            template.render({
-                'user_name': username,
-                'total_games': len(games),
-                'total_wins': sum(x[1][0] for x in matchup_data),
-                'total_points': total_points,
-                'matchup_data': matchup_data}))
+        content = template.render({
+            'user_name': username,
+            'total_games': len(games),
+            'total_wins': sum(x[1][0] for x in matchup_data),
+            'total_points': total_points,
+            'matchup_data': matchup_data,
+            'logout_url': usersapi.create_logout_url(self.request.path),
+            'is_admin_user': usersapi.is_current_user_admin()
+        })
+        self.response.write(content)
```

Comment:
Agreed; I didn't do a very good job of keeping the diff minimal on this PR. Will try to do that in the future.